routing/ROUTING_V1_HTTP.md

```diff
@@ -1,175 +1,3 @@
-- `429` (Too Many Requests): may be returned along with optional [Retry-After](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) header to indicate to the caller that it is issuing requests too quickly
-- `400` (Bad Request): must be returned if an unknown path is requested
-
-## CORS and Web Browsers
-
-Browser interoperability requires implementations to support
-[CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
-
-JavaScript client running on a third-party Origin must be able to send HTTP
-request to the endpoints defined in this specification, and read the received
-values. This means HTTP server implementing this API must (1) support
-[CORS preflight requests](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request)
-sent as HTTP OPTIONS, and (2) always respond with headers that remove CORS
-limits, allowing every site to query the API for results:
-
-```plaintext
-Access-Control-Allow-Origin: *
-Access-Control-Allow-Methods: GET, OPTIONS
-```
-
-## Known Transfer Protocols
-
-This section contains a non-exhaustive list of known transfer protocols (by name) that may be supported by clients and servers.
-
-### Bitswap
-
-Multicodec name: `transport-bitswap`
-Schema: `bitswap`
-Specification: [ipfs/specs/BITSWAP.md](https://github.com/ipfs/specs/blob/main/BITSWAP.md)
-
-#### Bitswap Read Provider Records
-
-```json
-{
-    "Protocol": "transport-bitswap",
-    "Schema": "bitswap",
-    "ID": "12D3K...",
-    "Addrs": ["/ip4/..."]
-}
-```
-
-- `ID`: the [Peer ID](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md) to contact
-- `Addrs`: a list of known multiaddrs for the peer
-  - This list may be incomplete or incorrect and should only be treated as *hints* to improve performance by avoiding extra peer lookups
-
-The server should respect a passed `transport` query parameter by filtering against the `Addrs` list.
-
-### Filecoin Graphsync
-
-Multicodec name: `transport-graphsync-filecoinv1`
-Schema: `graphsync-filecoinv1`
-Specification: [ipfs/go-graphsync/blob/main/docs/architecture.md](https://github.com/ipfs/go-graphsync/blob/main/docs/architecture.md)
-
-#### Filecoin Graphsync Read Provider Records
-
-```json
-{
-    "Protocol": "transport-graphsync-filecoinv1",
-    "Schema": "graphsync-filecoinv1",
-    "ID": "12D3K...",
-    "Addrs": ["/ip4/..."],
-    "PieceCID": "<cid>",
-    "VerifiedDeal": true,
-    "FastRetrieval": true
-}
-```
-
-- `ID`: the peer ID of the provider
-- `Addrs`: a list of known multiaddrs for the provider
-- `PieceCID`: the CID of the [piece](https://spec.filecoin.io/systems/filecoin_files/piece/#section-systems.filecoin_files.piece) within which the data is stored
-- `VerifiedDeal`: whether the deal corresponding to the data is verified
-- `FastRetrieval`: whether the provider claims there is an unsealed copy of the data available for fast retrieval
+Moved to https://specs.ipfs.tech/routing/routing-v1/
```

Comment:
@hacdias just to make this bit more future proof, let's make it clear in the `ref` and URL that this is HTTP API:

```suggestion
Moved to https://specs.ipfs.tech/routing/http-routing-v1/
```